ipa-core/src/bin/report_collector.rs

```diff
@@ -329,6 +342,55 @@ fn write_ipa_output_file(
     Ok(())
 }
 
+async fn hybrid(
+    args: &Args,
+    hybrid_query_config: HybridQueryParams,
+    helper_clients: &[IpaHttpClient<Helper>; 3],
```

Comment:
this does not seem right - we need to have a client per shard. Sharded shuffle in `test_mpc` may provide a way to structure this for hybrid